src/eq_schema/Answer.js

```diff
@@ -10,6 +10,10 @@ class Answer {
     this.label = answer.label;
     this.description = answer.description;
 
+    if (!isNil(answer.parentAnswerId)) {
```

Comment:
`parentAnswerId` is not exposed by the API. But you can work it out by checking the `other` field when building the answers. i.e. something like this. But this may have to happen at a level above (in Question.js)

```js
if (answer.other) {
  const parentAnswer = new Answer(answer);
  const childAnswer = new Answer(answer.other.answer);
  childAnswer.parent_answer_id = parentAnswer.id;
  parentAnswer.child_answer_id = childAnswer.id;
  answers.push(parentAnswer);
  answers.push(childAnswer);
}
```

Reply:
This `parentAnswerId` isn't actually referring to the api, when the child answer object is created I add in a new field named `parentAnswerId`.